src/meta/ClusterInfo.cpp

```diff
@@ -356,18 +353,21 @@ void ClusterInfo::load(const std::string& file, CreateMetaDB createDb) {
       name = td["topic"].as<std::string>();
     }
 
+    // hour table level retention.max-hr as single way to ingest and evict data
+    const auto retentionSeconds = (td["retention"]["max-hr"].as<double>() * Evidence::HOUR_SECONDS);
```

Comment:
nit: can we use a temporary var to capture td["retention"] for reuse?

Reply:
yes,